project/SparkBuild.scala

```diff
@@ -230,6 +231,8 @@ object SparkBuild extends PomBuild {
           // see `scalac -Wconf:help` for details
           "-Wconf:cat=deprecation:wv,any:e",
           // 2.13-specific warning hits to be muted (as narrowly as possible) and addressed separately
+          // TODO Enable this option when Scala 2.12 is no longer supported.
```

Comment:
@HyukjinKwon  A little interesting:

![image](https://user-images.githubusercontent.com/1475305/99783476-6d3f1f80-2b55-11eb-97ac-fb70d4c07f7b.png)

Scala 2.13 think this is a `unused import` but Scala 2.12 not. so comments `-Wunused:imports` in Scala 2.13 and left a TODO

Reply:
Can you file a JIRA under the Scala 2.13 umbrella JIRA? You can make a comment like `TODO(SPARK-XXXXX): blah blah`.

Reply:
OK

Reply:
Address 911acfe fix this